src/styles.css

```diff
@@ -18,6 +18,10 @@ body {
   display: grid
 }
 
+.hidden {
```

Comment:
We might want to consider getting rid of this in order to update our data model first, then the dom